src/main/kotlin/org/rust/cargo/toolchain/components/Cargo.kt

```diff
@@ -48,6 +52,14 @@ import java.nio.file.Files
 import java.nio.file.Path
 import java.nio.file.Paths
 
+fun RsToolchain.rawCargo(): Cargo = Cargo(this)
```

Comment:
Do we really need `raw` prefix here? I have never understood why it's needed. It looks rather inconsistent with other tools and doesn't add additional info (at least for me). Maybe drop it while this refactoring as well?